### DISCUSSION TITLE
>> More Link Functionality Override Needed

### DISCUSSION THREAD

**Discussion by janeofalltrades** [bluecaret/carettab#380] [Help and Support]
I recently began seeing in my Quick Links that if there are fewer than 7 links in the list I get the >> More link. This seems opposite of what someone would really want. Also, I don't want to use a >> More link and would like the ability to disable that feature so that all my links always show.
![Screenshot 2024-08-06 at 10 22 07 AM](https://github.com/user-attachments/assets/017a11b3-54c2-4045-8182-0d3d4f3bfdd4)

**Comment by bluecaret** [comment]
The more link should display if the amount of links are too much to show in the space provided. I do get your issue though, it seems here the only reason it is showing the last link in more is because the more button itself took up the space which is incorrect. I'm not spending much time on this extension at the moment so I can't say it will be fixed anytime soon, but I will add it to my list of things to work on.

For now a workaround would be to just make the container a bit taller till the last link displays. I know from an aesthetic sense that's not ideal but it is all I can suggest for now. 

Thank you for letting me know!

**Comment by janeofalltrades** [comment]
Thanks so much for your response! I totally was not expecting one. 🙂 Just want to make sure it is clear what the issue I'm experiencing is. On my original configuration the height is set to auto. Since your message, I tried the trick of making the container taller, both by pixels and percent, but neither caused the >> More tag to go away, even set at 100%. It only goes away after I add the 7th link. Interestingly, until the 7th link is added, it only prevents me from seeing whatever the last link is. So, if I have 4 links, I can see the first 3 and get the >> More tag which reveals the 4th and last link when clicked. It may be helpful to know that I have 6 different quick links widgets, so not sure if that is a factor.

I'm super excited to use your app and realize you are not supporting it at the same level anymore, but figured I'd throw this into the ring for consideration when you are doing something on it again. Thanks so much for contributing! If I can help, let me know.

Thanks,
Amy (Kuenzel) Wallace
________________________________

**Reply by bluecaret** [reply]
Thanks for the clarification! That will help a lot and ya sounds like something is not right for sure :)

**Comment by bluecaret** [comment]
I believe I got this fixed and just pushed version 4.6.0. It should be live in a few days. Thank you!
